react-native-libraries.json

```diff
@@ -5861,5 +5861,15 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/oblador/react-native-performance/tree/master/packages/react-native-performance",
+    "examples": ["https://github.com/oblador/react-native-performance/tree/master/examples/vanilla"],
+    "images": [
+      "https://user-images.githubusercontent.com/378279/105892056-9f677480-6011-11eb-895a-f3f8653449c8.png"
+    ],
+    "ios": true,
+    "android": true,
+    "expo": true
```

Comment:
I'm a bit unsure about this one. The library has a native component that is not available on expo, but it works without it.

Reply:
It looks like your example converted to Expo Snack works fine (https://snack.expo.io/@simek/github.com-oblador-react-native-performance:examples-vanilla), so I see no reason not to mark Expo as supported. 

We can always revert it, if there will be some issues reported from Expo users. 🙂